understanding/redundant-entry.html

```diff
@@ -56,191 +55,142 @@
-                  implementation (for meeting this criterion) could leak additional PII.
+               <p>この達成基準には、プライバシー又は個人を特定できる情報 (PII) に特有の要件又は例外は含まれていないが、自動入力などの技術を実装する場合、コンテンツ制作者は、プロセス中に一時的であっても情報を保存する際にデータ保護を確保すべきである。追加のプライバシーリスクをもたらさない方法で冗長な入力項目を排除することは可能であるが、(この達成基準を満たすための) 不十分な実装によって追加の PII が漏洩する可能性もある。
                </p>
-               <p>There are exceptions for:</p>
+               <p>次の場合は例外である:</p>
                <ul>
                   
-                  <li>Essential uses of input re-entry for things like memory games which would be invalidated
-                     if the previous answers were supplied.
+                  <li>記憶ゲームなど、前の答えが提供されると無効になってしまうような、入力項目の再入力が必要不可欠な用途。
                   </li>
                   
-                  <li>Security measures such as preventing a password string from being shown or copied.
-                     When creating a password, it should be a unique and complex string and therefore cannot
-                     be validated by the author. If the system requires the user to manually create a password
-                     that is not displayed, having users re-validate their new string is allowed as an
-                     exception.
+                  <li>パスワード文字列が表示又はコピーされないようにするなどのセキュリティ対策。パスワードを作成するとき、パスワードは一意で複雑な文字列である必要があるため、コンテンツ制作者は検証できない。システムが、表示されないパスワードを利用者が手動で作成することを要求する場合、利用者に新しい文字列を再検証させることは例外として許可される。
                   </li>
                   
-                  <li>When the previously entered information is no longer valid, it can be requested that
-                     the user enter that information again.
+                  <li>以前に入力された情報が有効でなくなった場合、利用者にその情報を再度入力するように要求できる。
                   </li>
                   
                </ul>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
+               <h2>メリット</h2>
                <ul>
                   
-                  <li>Users with cognitive disabilities experience short-term, working memory difficulty.
-                     Not having to repeatedly remember particular information reduces stress and the likelihood
-                     of mistakes.
+                  <li>認知障害のある使用者は、短期的な作業記憶の難しさを感じている。特定の情報を繰り返し思い出す必要がないため、ストレスが軽減され、間違いが起こりにくくなる。
                   </li>
                   
-                  <li>Users who experience difficulty forming new memories, recalling information, and other
-                     functions related to cognition can complete processes without having to unnecessarily
-                     rely on their memory.
+                  <li>新しい記憶の形成、情報の想起、および認知に関連するその他の機能に難しさを感じている利用者は、必要以上に記憶に頼ることなくプロセスを完了できる。
                   </li>
                   
-                  <li>Users with mobility impairments, for example using switch control or voice input,
-                     benefit from a reduced need for text entry.
+                  <li>例えば、スイッチ操作、音声入力を使用するような、運動障害のある利用者は、テキストの入力項目の必要性が軽減されるというメリットがある。
```

Comment:
「入力項目の必要性」は、たしかに原文では「entry」ですが、ここでは「入力の必要性」なのかな、という気がしました。

```suggestion
                  <li>例えば、スイッチ操作、音声入力を使用するような、運動障害のある利用者は、テキストの入力の必要性が軽減されるというメリットがある。
```